cli/certs/certs.go

```diff
@@ -166,3 +174,19 @@ func newCertificate(org string) (*x509.Certificate, error) {
 		BasicConstraintsValid: true,
 	}, nil
 }
+
+func ReadCerts(caCertFileLocation, certFileLocation, certKeyLocation string) (*certList, error) {
+	caCertFile, err := os.ReadFile(caCertFileLocation)
+	if err != nil {
+		return nil, err
+	}
+	certFile, err := os.ReadFile(fmt.Sprintf(certFileLocation))
```

Comment:
do we need `fmt.Sprintf(certFileLocation)`

Reply:
done , good catch